### PR TITLE
Delete SDL logo related information

### PR DESCRIPTION
**Pull request by undef21**

### CODE REVIEW

**Comment by ihhub** [comment]
Hi @undef21 , thanks for cleaning up the project. We're still hesitating about fully excluding this logo from the project. We need to make sure that using SDL itself doesn't require to show a logo. If no then we're good to go.

File: src/fheroes2/game/fheroes2.cpp [hunk 1]
```diff
@@ -41,7 +41,6 @@
 #include "test.h"
 #include "zzlib.h"
```
Comment (ihhub) [inline src/fheroes2/game/fheroes2.cpp:42]: Hi @undef21 , could you please check whether we need this header after these changes?

Comment (undef21) [reply src/fheroes2/game/fheroes2.cpp:42]: Hi @ihhub, "zzlib.h" is needed for fheroes2.cpp:155 (load app icon).
"test.h" is unneeded. Also "test.h" is used in game.cpp for tests, which are run by './fheroes -t3' cmd in DEBUG build. If tests are not used, may be delete them?

**Comment by ihhub** [comment]
@undef21 , thanks a lot for removing this unused part of code!

**Pull request merged**